src/app/Fake.Azure.CloudServices/CloudServices.fs

```diff
@@ -0,0 +1,97 @@
+﻿/// Contains tasks to package Azure Cloud Services.
+module Fake.Azure.CloudServices
+
+open System.IO
+open Fake.Core
+open Fake.IO
+open Fake.IO.Globbing.Operators
+
+/// Configuration details for packaging cloud services.
+[<CLIMutable>]
+type PackageCloudServiceParams =
+    { /// The name of the Cloud Service.
+      CloudService : string
+      /// The name of the role in the service.
+      WorkerRole : string
+      /// The SDK version to use e.g. 2.2. If None, the latest available version is used.
+      SdkVersion : float option
```

Comment:
Version is a float?

Reply:
I copied all this from the `FakeLib` project.